From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 28 Jan 2020 11:09:14 -0600
Subject: [PATCH] fix drm/i915: Move rps.hw_lock to dev_priv

The file was
0998-UPSTREAM-drm-i915-Move-rps.hw_lock-to-dev_priv-and-s.patch
---
 drivers/gpu/drm/i915/intel_pm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index eab65e549..8b2073e79 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -8205,9 +8205,9 @@ static void __intel_disable_rc6(struct drm_i915_private *dev_priv)
 
 static void intel_disable_rc6(struct drm_i915_private *dev_priv)
 {
-	mutex_lock(&dev_priv->rps.hw_lock);
+	mutex_lock(&dev_priv->pcu_lock);
 	__intel_disable_rc6(dev_priv);
-	mutex_unlock(&dev_priv->rps.hw_lock);
+	mutex_unlock(&dev_priv->pcu_lock);
 }
 
 static void intel_disable_rps(struct drm_i915_private *dev_priv)
-- 
https://clearlinux.org

